src/common/components/signUpModal/signUpModal.component.js

```diff
@@ -1,17 +1,20 @@
 import angular from 'angular'
 import includes from 'lodash/includes'
+import OktaSignIn from '@okta/okta-signin-widget'
 import sessionService, { Roles } from 'common/services/session/session.service'
 import orderService from 'common/services/api/order.service'
 import template from './signUpModal.tpl.html'
 import cartService from 'common/services/api/cart.service'
+require('assets/okta-sign-in/css/okta-sign-in.min.css')
```

Comment:
We can add it to `scss` folder, but this would mean each time we update the package, we would also need to update the `scss` file manually. - I prefer not to have any manual work of updating `scss` files when updating the Okta package but instead allow Webpack to manage to update it.

However, I might not see something, so feel free to disagree.

Reply:
I think it might warrant a comment in the code. Theoretically it could be made part of the build process, but probably not worth the effort.